storage/httpstore.go

```diff
@@ -39,7 +39,14 @@ type NetworkError struct {
 }
 
 func (n NetworkError) Error() string {
-	return n.Wrapped.Error()
```

Comment:
Following up on https://github.com/docker/notary/issues/1007#issuecomment-254635035:

Nonblocking:  In that thread I was suggesting that rather that we can check to see if ``n.Wrapped` is a `url.Error` and if so, convert the wrapped error to a new `url.Error` with the same params but with the URL escaped - that way we know that we'd always be applying the unescape specifically to a URL, and not some other string that might have an arbitrary `%` in it.

I don't know of any other network errors that have a `%` though - this is likely fine as it is, and is just my being pedantic.